ci/toolchain_manager.sh

```diff
@@ -52,6 +52,7 @@ function deploy {
         FORCED_TAG="${TAG}" \
         QUAY_NAMESPACE="${QUAY_NAMESPACE}" \
         SECOND_MEMBER_MODE=false \
+        DEPLOY_LATEST=true \
```

Comment:
I remember these flag to also configure the installed operators in such a way it is using the `latest` version. Is this going to impact on what we deploy? Should we propose to use a dedicated flag for skipping pairing, like `CI_SKIP_PAIRING` instead?

Reply:
Although, oddly enough, our e2e tests pass when I run against this version of kubesaw.

Reply:
https://github.com/codeready-toolchain/toolchain-e2e/pull/1034#issuecomment-2297326327

Reply:
As of 0e508de, we now use the newly-introduced `CI_DISABLE_PAIRING` parameter.